Formula/vsftpd.rb

```diff
@@ -44,11 +44,45 @@ def caveats
       s += <<-EOS.undent
         vsftpd was compiled with SSL support. To use it you must generate a SSL
         certificate and set 'enable_ssl=YES' in your config file.
+
       EOS
     end
 
+    s += <<-EOS.undent
+      vsftpd requires root privileges in most cases so you will need to run
```

Comment:
Technically, it is possible to run vsfptd as a user other than root, however, doing so is less secure. From `vsftpd.conf(5)`:
```
run_as_launching_user
       Set to YES if you want vsftpd to run as the user which launched  vsftpd.  This  is  useful  where  root
       access  is  not  available. MASSIVE WARNING! Do NOT enable this option unless you totally know what you
       are doing, as naive use of this option can create massive security problems. Specifically, vsftpd  does
       not / cannot use chroot technology to restrict file access when this option is set (even if launched by
       root). A poor substitute could be to use a deny_file setting such as {/*,*..*}, but the reliability  of
       this cannot compare to chroot, and should not be relied on.  If using this option, many restrictions on
       other options apply. For example, options requiring privilege such as non-anonymous logins, upload own-
       ership  changing,  connecting  from  port  20 and listen ports less than 1024 are not expected to work.
       Other options may be impacted.
```
I will update the caveat to make it clear that running as a user other than root is not recommended.